azure-pipelines.yml

```diff
@@ -41,6 +41,12 @@ jobs:
             version: $(DotNetVersion)
         - pwsh: |
             dotnet workload install android
+        - task: JavaToolInstaller@0
+          inputs:
+            versionSpec: '11'
+            jdkArchitectureOption: 'x64'
+            jdkSourceOption: 'PreInstalled'
```

Comment:
What was the error before adding this? We might look into it.

Was the issue that `$JAVA_HOME` points to some other JDK we don't want?

Reply:
```
error XA0031: Java SDK 11.0 or above is required when using .NET 6 or higher.
```